lib/song.dart

```diff
@@ -207,7 +210,7 @@ class _SongState extends State<Song> {
           maxWidth: MediaQuery.of(context).size.width < 600
               ? MediaQuery.of(context).size.width
               : MediaQuery.of(context).size.width * 0.6,
-          maxHeight: MediaQuery.of(context).size.height * 0.6),
+          maxHeight: MediaQuery.of(context).size.height * 0.7),
```

Comment:
Ok, these hardcoded numbers are bad. This was a hack.

But any reason why you are particularly changing 0.6 to 0.7 here?

Reply:
I played around with it and I found that this value showed a bit more of the bottom of the list, partially showing the last collection. This helps for some devices to indicate that it is scrollable (like mine).
 
I've tried to improve it slightly by calculating the height and width once and then using a Provider to pass it down the widget tree. LMK what you think of it